definitions/scenarios/maintenance_mode.rb

```diff
@@ -0,0 +1,44 @@
+module ForemanMaintain::Scenarios
+  class MaintenanceModeStart < ForemanMaintain::Scenario
+    metadata do
+      description 'Start maintenance-mode'
+      tags :maintenance_mode_start
+      label :maintenance_mode_start
+      manual_detection
+    end
+
+    def compose
+      add_step(Procedures::Iptables::AddChain.new)
+      add_step(Procedures::Cron::Stop.new)
```

Comment:
Yes, doing these changes for `crond` service.

Just to confirm that to achieve this, I have to change logic inside [service](https://github.com/theforeman/foreman_maintain/blob/master/definitions/features/service.rb#L10) feature because using `katello-service`, actions can be performed on fix set of services not on all services. So it would be good to check list from katello-service and use else block otherwise which uses either `service-wait` or `systemctl`.